packages/combobox/src/ComboboxMenu/Menu.styles.ts

```diff
@@ -109,7 +132,16 @@ export const menuMessageSizeStyle: Record<Size, string> = {
   `,
 };
 
+// TODO: this is not used
 export const menuMessageIconSizeStyle: Record<Size, string> = {
+  [Size.XSmall]: css`
+    height: ${typeScales.body1.fontSize}px;
+    width: ${typeScales.body1.fontSize}px;
+  `,
+  [Size.Small]: css`
+    height: ${typeScales.body1.fontSize}px;
+    width: ${typeScales.body1.fontSize}px;
+  `,
```

Comment:
This object could also be a function if the typesScales.body1 reference was set in a const